src/main/java/step3/domain/RacingCar.java

```diff
@@ -20,13 +22,18 @@ public String getCarName() {
     }
 
     public void move(int power) {
-        if(checkPowerThreshold(power)) {
+        if (checkPowerThreshold(power)) {
             this.currentPosition += MOVE_DISTANCE;
         }
     }
 
     private boolean checkPowerThreshold(int power) {
         return power >= POWER_THRESHOLD;
     }
+
+    public boolean samePosition (int maxPosition) {
+        return (currentPosition == maxPosition) ? true : false;
```

Comment:
```suggestion
        return currentPosition == maxPosition
```
가독성 향상을 위해서 표시해시는걸 까요 ?

Reply:
습관이.. 피드백 주신데로 고치겠습니다.